projects/hr-portal/src/hr-portal.ts

```diff
@@ -1,83 +1,195 @@
+
+        <igc-grid-toolbar>
+          <igc-grid-toolbar-title>HR Portal</igc-grid-toolbar-title>
+          <igc-grid-toolbar-actions>
+            <igc-grid-toolbar-hiding></igc-grid-toolbar-hiding>
+            <igc-grid-toolbar-pinning></igc-grid-toolbar-pinning>
+            <igc-grid-toolbar-exporter>
+              <span slot="excelText">Export</span>
+            </igc-grid-toolbar-exporter>
+            <igc-grid-toolbar-advanced-filtering></igc-grid-toolbar-advanced-filtering>
+          </igc-grid-toolbar-actions>
+        </igc-grid-toolbar>
+
+        <igc-column field="Name" width="300px" ?sortable="true" pinned="true" .bodyTemplate="${this.avatarTemplate}"></igc-column>
+        <igc-column field="JobTitle" header="Job Title" data-type="string" min-width="200px" sortable="true"></igc-column>
+        <igc-column field="Department" data-type="string" min-width="200px" sortable="true"></igc-column>
+
+        <igc-column field="Location" data-type="string" ?sortable="true" .bodyTemplate="${this.countryIconTemplate}"></igc-column>
+        <igc-column field="Contacts" data-type="string" min-width="200px" filterable="false" .bodyTemplate="${this.contactsTemplate}"></igc-column> 
 
+        <igc-column field="HireDate" header="Hire Date" data-type="date" min-width="100px" sortable="true" .bodyTemplate="${this.dateTemplate}"></igc-column>
+
+        <igc-column field="GrossSalary" header="Gross Salary" data-type="currency" min-width="100px" sortable="true"></igc-column></igc-column>
+      </igc-tree-grid>
+    `;
+  }
 
   static styles = css`
-    :host {
-      max-width: 1280px;
+  :host {
+      display: block;
       margin: 0 auto;
-      padding: 2rem;
+      align-items: center;
       text-align: center;
-    }
+      justify-content: center;
+      gap: 8px;
+  }
 
-    .logo {
-      height: 6em;
-      padding: 1.5em;
-      will-change: filter;
-      transition: filter 300ms;
-    }
-    .logo:hover {
-      filter: drop-shadow(0 0 2em #646cffaa);
-    }
-    .logo.lit:hover {
-      filter: drop-shadow(0 0 2em #325cffaa);
-    }
+  .gridStyle {
+      --ig-size: var(--ig-size-medium);
+      --row-even-background: #ffffff;
+  }
+  
+  .small {
+    --ig-size: var(--ig-size-small);
+  }
 
-    .card {
-      padding: 2em;
-    }
+  .center-content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    height: 100%;
+    width: 100%;
+    gap: 8px;
+  }
 
-    .read-the-docs {
-      color: #888;
-    }
+  .employeeDiv {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+  }
 
-    ::slotted(h1) {
-      font-size: 3.2em;
-      line-height: 1.1;
-    }
+  igc-avatar::part(base) {
+    --size: 28px;
+  }
 
-    a {
-      font-weight: 500;
-      color: #646cff;
-      text-decoration: inherit;
-    }
-    a:hover {
-      color: #535bf2;
-    }
+  igc-tree-grid::part(row) {
+      background-color: #ffffff(--row-even-background);
```

Comment:
I think this is not used. Also, it is not a correct value for `background-color`

Reply:
it applies different coloring for even rows, same as in angular.  Not very visible on some displays but it does apply.